DOCS-2319 Synthetic Monitoring API Test, Browser Test, and Multistep API Test Edits (#11433)
DataDog/documentation: content/en/synthetics/browser_tests/_index.md

<div class="alert alert-info">Are you interested in testing applications that sit behind MFA? Read <a href="/synthetics/guide/app-that-requires-login/#multi-factor-authentication" target="_blank">our dedicated guide </a> and <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjx8PDZ8kJ3MD2ehouTri9z_Fh7PoK90J8arRQgt7QFgFxog/viewform?usp=sf_link">send us feedback</a> to help us work on the systems that matter the most to your teams.</div>

## Permissions

> By default, only users with the [Datadog Admin and Datadog Standard roles][1] can create, edit, and delete Synthetic browser tests. To get create, edit, and delete access to Synthetic browser tests, upgrade your user to one of those [default roles][1].

Reviewer: ```suggestion
By default, only users with the [Datadog Admin and Datadog Standard roles][1] can create, edit, and delete Synthetic browser tests. To get create, edit, and delete access to Synthetic browser tests, upgrade your user to one of those two [default roles][1].
```

> By default, only users with the [Datadog Admin and Datadog Standard roles][1] can create, edit, and delete Synthetic browser tests. To get create, edit, and delete access to Synthetic browser tests, upgrade your user to one of these two [default roles][1].

Reviewer 2: ```suggestion
By default, only users with the [Datadog Admin and Datadog Standard roles][1] can create, edit, and delete Synthetic browser tests. To get create, edit, and delete access to Synthetic browser tests, upgrade your user to one of those [default roles][1].
```

> If you have access to the [custom role feature][2], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

Reviewer: ```suggestion
If you have access to the [custom role feature][2], add your user to any custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.
```

You can customize alert conditions to define the circumstances under which you want a test to send a notification alert.

* An alert is triggered if any assertion fails for `X` minutes from any `n` of `N` locations. This alerting rule allows you to specify for how much time and on how many locations a test needs to fail before triggering the notification.

> * Retry `X` times before location is marked as failed. This allows you to define how many consecutive test failures need to happen for a location to be considered as failed. By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][6].

Author: ```suggestion
* Retry `X` times before location is marked as failed. This allows you to define how many consecutive test failures need to happen for a location to be considered as failed. By default, there is a 300ms wait before retrying a test that failed. This interval can be configured with the [API][6].
```

    Notification messages include the **message** defined in this section and information about the failing locations.

2. Choose your [services][11] and/or team members to notify.
3. Specify a renotification frequency. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
4. Click **Save Details and Record Test**.

## Record your steps

> Tests can be only recorded from [Google Chrome][12]. To record your test, download the [Datadog Record Test extension for Google Chrome][13].

Reviewer 2: ```suggestion
Tests can only be recorded from [Google Chrome][12]. To record your test, download the [Datadog Record Test extension for Google Chrome][13].
```
